engine/yellow_intro.asm

```diff
@@ -125,20 +125,35 @@ ExecuteCurrentSceneSubroutine:
 	call LoadSceneSubroutineAddress
 	jp hl
 
+; Super-uber hacky very temporary solution for disabling scenes--alignment matters here, too tired to look into it right now.
+; Will be fixed soon™️®️
```

Comment:
Putting a comment here to remind myself

Reply:
Issue seems to stem from `InitYellowIntroGFXAndMusic`. Looking inside, it looks like there are some raw addresses used. Figures.

Reply:
Each word added seems to lower the loaded sprites by a pixel.

Reply:
Responsible lines [here](https://github.com/DarkSideHunterX/pokepink/blob/318ac3b4760c421b32c2bf0271498bd4508010c8/engine/yellow_intro.asm#L836-L843)
Will work on a fix later.

Reply:
I think that the addresses point to the `INCLUDE`'d graphics in [bank3e.asm](https://github.com/DarkSideHunterX/pokepink/blob/master/engine/bank3e.asm).

Reply:
Fixed.